{
  "_feedstock_status": {
    "acctaudem": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "clean",
      "pr_url": "https://github.com/conda-forge/acctaudem-feedstock/pull/13",
      "pre_pr_migrator_status": ""
    },
    "autorift": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "unstable",
      "pr_url": "https://github.com/conda-forge/autorift-feedstock/pull/49",
      "pre_pr_migrator_status": ""
    },
    "dggrid": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "clean",
      "pr_url": "https://github.com/conda-forge/dggrid-feedstock/pull/5",
      "pre_pr_migrator_status": ""
    },
    "exactextract": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "clean",
      "pr_url": "https://github.com/conda-forge/exactextract-feedstock/pull/4",
      "pre_pr_migrator_status": ""
    },
    "fiona": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9139802326'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_ppc64le_numpy1.22python3.10.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.22python3.8.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.22python3.9.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.23python3.11.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.26python3.12.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).\n                </pre>\n                </div>\n                </details>"
    },
    "gazebo": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "",
      "pr_url": "https://github.com/conda-forge/gazebo-feedstock",
      "pre_pr_migrator_status": ""
    },
    "gdal-csharp": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "",
      "pr_url": "https://github.com/conda-forge/gdal-csharp-feedstock",
      "pre_pr_migrator_status": ""
    },
    "gmt": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9143944538'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_ppc64le_: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 gdal 3.9.*  does not exist (perhaps a typo or a missing channel).\n                </pre>\n                </div>\n                </details>"
    },
    "gz-common": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9143403814'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_ppc64le_ffmpeg6: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_ffmpeg7: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).\n                </pre>\n                </div>\n                </details>"
    },
    "heavydb-ext": {
      "immediate_children": [
        "omniscidb"
      ],
      "num_descendants": 1,
      "pr_status": "unstable",
      "pr_url": "https://github.com/conda-forge/heavydb-ext-feedstock/pull/72",
      "pre_pr_migrator_status": ""
    },
    "helios": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "",
      "pr_url": "https://github.com/conda-forge/helios-feedstock",
      "pre_pr_migrator_status": ""
    },
    "hexer": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "clean",
      "pr_url": "https://github.com/conda-forge/hexer-feedstock/pull/14",
      "pre_pr_migrator_status": ""
    },
    "isce2": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "unstable",
      "pr_url": "https://github.com/conda-forge/isce2-feedstock/pull/68",
      "pre_pr_migrator_status": ""
    },
    "isce3": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "unstable",
      "pr_url": "https://github.com/conda-forge/isce3-feedstock/pull/61",
      "pre_pr_migrator_status": ""
    },
    "lue": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "unstable",
      "pr_url": "https://github.com/conda-forge/lue-feedstock/pull/47",
      "pre_pr_migrator_status": ""
    },
    "mapserver": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9140537557'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_ppc64le_python3.10.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_python3.11.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_python3.12.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_python3.8.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_python3.9.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).\n                </pre>\n                </div>\n                </details>"
    },
    "mdal": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "",
      "pr_url": "https://github.com/conda-forge/mdal-feedstock",
      "pre_pr_migrator_status": ""
    },
    "mesher": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "",
      "pr_url": "https://github.com/conda-forge/mesher-feedstock",
      "pre_pr_migrator_status": ""
    },
    "nansat": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "unstable",
      "pr_url": "https://github.com/conda-forge/nansat-feedstock/pull/40",
      "pre_pr_migrator_status": ""
    },
    "ncl": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "",
      "pr_url": "https://github.com/conda-forge/ncl-feedstock",
      "pre_pr_migrator_status": ""
    },
    "pcraster": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "",
      "pr_url": "https://github.com/conda-forge/pcraster-feedstock",
      "pre_pr_migrator_status": ""
    },
    "pdal": {
      "immediate_children": [
        "pdal_wrench",
        "qgis"
      ],
      "num_descendants": 2,
      "pre_pr_migrator_status": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9142170238'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_ppc64le_: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 geotiff 1.7.1.*  is installable with the potential options\n\u2502  \u251C\u2500 geotiff 1.7.1, which can be installed;\n\u2502  \u251C\u2500 geotiff 1.7.1 would require\n\u2502  \u2502  \u2514\u2500 proj [>=9.0.1,<9.0.2.0a0 |>=9.2.0,<9.2.1.0a0 |>=9.2.1,<9.2.2.0a0 |>=9.3.1,<9.3.2.0a0 ] with the potential options\n\u2502  \u2502     \u251C\u2500 proj 9.2.0 would require\n\u2502  \u2502     \u2502  \u2514\u2500 proj4 999999999999 , which can be installed;\n\u2502  \u2502     \u251C\u2500 proj 9.2.1 would require\n\u2502  \u2502     \u2502  \u2514\u2500 proj4 999999999999 , which can be installed;\n\u2502  \u2502     \u251C\u2500 proj 9.3.1 would require\n\u2502  \u2502     \u2502  \u2514\u2500 proj4 999999999999 , which can be installed;\n\u2502  \u2502     \u2514\u2500 proj 9.0.1 would require\n\u2502  \u2502        \u2514\u2500 proj4 999999999999 , which can be installed;\n\u2502  \u2514\u2500 geotiff 1.7.1 would require\n\u2502     \u2514\u2500 proj >=9.4.0,<9.4.1.0a0 , which does not exist (perhaps a missing channel);\n\u251C\u2500 hdf5 1.14.3.*  is installable with the potential options\n\u2502  \u251C\u2500 hdf5 1.14.3, which can be installed;\n\u2502  \u2514\u2500 hdf5 1.14.3 would require\n\u2502     \u2514\u2500 openssl >=3.3.0,<4.0a0 , which can be installed;\n\u251C\u2500 libgdal is installable with the potential options\n\u2502  \u251C\u2500 libgdal 3.7.3 would require\n\u2502  \u2502  \u251C\u2500 libxml2 >=2.12.2,<3.0.0a0 , which requires\n\u2502  \u2502  \u2502  \u2514\u2500 icu >=73.2,<74.0a0 , which can be installed;\n\u2502  \u2502  \u251C\u2500 proj >=9.3.1,<9.3.2.0a0 , which cannot be installed (as previously explained);\n\u2502  \u2502  \u2514\u2500 tiledb >=2.18.2,<2.19.0a0  with the potential options\n\u2502  \u2502     \u251C\u2500 tiledb 2.18.2, which can be installed;\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.0] would require\n\u2502  \u2502     \u2502  \u2514\u2500 libcurl >=7.87.0,<8.0a0 , which can be installed;\n\u2502  \u2502     \u251C\u2500 tiledb [2.16.3|2.18.2] would require\n\u2502  \u2502     \u2502  \u2514\u2500 openssl >=1.1.1w,<1.1.2a , which conflicts with any installable versions previously reported;\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.1] would require\n\u2502  \u2502     \u2502  \u2514\u2500 libcurl >=7.88.1,<8.0a0 , which can be installed;\n\u2502  \u2502     \u2514\u2500 tiledb [2.18.3|2.18.4], which can be installed;\n\u2502  \u251C\u2500 libgdal [3.2.2|3.3.0|...|3.4.0] would require\n\u2502  \u2502  \u2514\u2500 libpq >=13.3,<14.0a0  with the potential options\n\u2502  \u2502     \u251C\u2500 libpq [13.2|13.3] would require\n\u2502  \u2502     \u2502  \u2514\u2500 krb5 >=1.19.1,<1.20.0a0 , which can be installed;\n\u2502  \u2502     \u251C\u2500 libpq [12.9|13.3|13.5] would require\n\u2502  \u2502     \u2502  \u2514\u2500 krb5 >=1.19.2,<1.20.0a0 , which can be installed;\n\u2502  \u2502     \u2514\u2500 libpq 13.8 would require\n\u2502  \u2502        \u2514\u2500 krb5 >=1.19.3,<1.20.0a0 , which can be installed;\n\u2502  \u251C\u2500 libgdal 3.2.2 would require\n\u2502  \u2502  \u2514\u2500 libpq >=13.1,<14.0a0  with the potential options\n\u2502  \u2502     \u251C\u2500 libpq [13.1|13.2] would require\n\u2502  \u2502     \u2502  \u2514\u2500 krb5 >=1.17.1,<1.18.0a0 , which can be installed;\n\u2502  \u2502     \u251C\u2500 libpq [13.2|13.3], which can be installed (as previously explained);\n\u2502  \u2502     \u251C\u2500 libpq [12.9|13.3|13.5], which can be installed (as previously explained);\n\u2502  \u2502     \u2514\u2500 libpq 13.8, which can be installed (as previously explained);\n\u2502  \u251C\u2500 libgdal [3.3.3|3.4.0|3.4.1|3.4.2] would require\n\u2502  \u2502  \u2514\u2500 geotiff >=1.7.0,<1.7.1.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u251C\u2500 libgdal [3.4.2|3.4.3|3.5.0] would require\n\u2502  \u2502  \u251C\u2500 hdf5 >=1.12.1,<1.12.2.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2514\u2500 proj >=9.0.0,<9.0.1.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u251C\u2500 libgdal [3.5.0|3.5.1] would require\n\u2502  \u2502  \u251C\u2500 hdf5 >=1.12.1,<1.12.2.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2514\u2500 proj >=9.0.1,<9.0.2.0a0 , which cannot be installed (as previously explained);\n\u2502  \u251C\u2500 libgdal [3.5.1|3.5.2|3.5.3|3.6.3] would require\n\u2502  \u2502  \u251C\u2500 hdf5 >=1.12.2,<1.12.3.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2514\u2500 proj [>=9.0.1,<9.0.2.0a0 |>=9.2.0,<9.2.1.0a0 ] with the potential options\n\u2502  \u2502     \u251C\u2500 proj 9.2.0, which can be installed (as previously explained);\n\u2502  \u2502     \u2514\u2500 proj 9.0.1, which cannot be installed (as previously explained);\n\u2502  \u251C\u2500 libgdal [3.4.3|3.5.2|...|3.6.2] would require\n\u2502  \u2502  \u251C\u2500 hdf5 >=1.12.2,<1.12.3.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2514\u2500 proj >=9.1.0,<9.1.1.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u251C\u2500 libgdal [3.5.3|3.6.1|3.6.2|3.6.3] would require\n\u2502  \u2502  \u251C\u2500 hdf5 >=1.12.2,<1.12.3.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2514\u2500 proj >=9.1.1,<9.1.2.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u251C\u2500 libgdal [3.5.3|3.6.3|3.6.4|3.7.0] would require\n\u2502  \u2502  \u251C\u2500 hdf5 >=1.14.0,<1.14.1.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2514\u2500 proj [>=9.2.0,<9.2.1.0a0 |>=9.2.1,<9.2.2.0a0 ] with the potential options\n\u2502  \u2502     \u251C\u2500 proj 9.2.0, which can be installed (as previously explained);\n\u2502  \u2502     \u2514\u2500 proj 9.2.1, which cannot be installed (as previously explained);\n\u2502  \u251C\u2500 libgdal [3.6.4|3.7.1|3.7.2] would require\n\u2502  \u2502  \u251C\u2500 icu >=73.2,<74.0a0 , which can be installed;\n\u2502  \u2502  \u251C\u2500 libpq >=15.4,<16.0a0  with the potential options\n\u2502  \u2502  \u2502  \u251C\u2500 libpq [15.4|15.5|15.6|15.7] would require\n\u2502  \u2502  \u2502  \u2502  \u2514\u2500 libxml2 [>=2.12.3,<3.0.0a0 |>=2.12.4,<3.0a0 |>=2.12.5,<3.0a0 |>=2.12.6,<3.0a0 ], which can be installed (as previously explained);\n\u2502  \u2502  \u2502  \u251C\u2500 libpq 15.4 would require\n\u2502  \u2502  \u2502  \u2502  \u2514\u2500 openssl >=3.1.2,<3.2.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2502  \u251C\u2500 libpq 15.4 would require\n\u2502  \u2502  \u2502  \u2502  \u2514\u2500 openssl >=3.1.3,<3.2.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2502  \u2514\u2500 libpq 15.4 would require\n\u2502  \u2502  \u2502     \u2514\u2500 openssl >=3.2.0,<3.2.0a0 , which does not exist (perhaps a missing channel);\n\u2502  \u2502  \u2514\u2500 proj >=9.2.1,<9.2.2.0a0 , which cannot be installed (as previously explained);\n\u2502  \u251C\u2500 libgdal [3.6.4|3.7.0|3.7.1] would require\n\u2502  \u2502  \u251C\u2500 hdf5 >=1.14.1,<1.14.2.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2514\u2500 proj >=9.2.1,<9.2.2.0a0 , which cannot be installed (as previously explained);\n\u2502  \u251C\u2500 libgdal [3.6.4|3.7.1] would require\n\u2502  \u2502  \u251C\u2500 icu >=72.1,<73.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u251C\u2500 libpq >=15.4,<16.0a0  with the potential options\n\u2502  \u2502  \u2502  \u251C\u2500 libpq [15.4|15.5|15.6|15.7], which can be installed (as previously explained);\n\u2502  \u2502  \u2502  \u251C\u2500 libpq 15.4, which cannot be installed (as previously explained);\n\u2502  \u2502  \u2502  \u251C\u2500 libpq 15.4, which cannot be installed (as previously explained);\n\u2502  \u2502  \u2502  \u2514\u2500 libpq 15.4, which cannot be installed (as previously explained);\n\u2502  \u2502  \u2514\u2500 proj >=9.2.1,<9.2.2.0a0 , which cannot be installed (as previously explained);\n\u2502  \u251C\u2500 libgdal [3.6.4|3.7.2] would require\n\u2502  \u2502  \u251C\u2500 libpq >=15.4,<16.0a0  with the potential options\n\u2502  \u2502  \u2502  \u251C\u2500 libpq [15.4|15.5|15.6|15.7], which can be installed (as previously explained);\n\u2502  \u2502  \u2502  \u251C\u2500 libpq 15.4, which cannot be installed (as previously explained);\n\u2502  \u2502  \u2502  \u251C\u2500 libpq 15.4, which cannot be installed (as previously explained);\n\u2502  \u2502  \u2502  \u2514\u2500 libpq 15.4, which cannot be installed (as previously explained);\n\u2502  \u2502  \u2514\u2500 proj >=9.3.0,<9.3.1.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u251C\u2500 libgdal [3.6.4|3.7.2|3.7.3|3.8.0|3.8.1] would require\n\u2502  \u2502  \u251C\u2500 proj >=9.3.0,<9.3.1.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2514\u2500 tiledb >=2.16,<2.17.0a0  with the potential options\n\u2502  \u2502     \u251C\u2500 tiledb [2.16.3|2.18.2], which cannot be installed (as previously explained);\n\u2502  \u2502     \u251C\u2500 tiledb [2.16.0|2.16.1|2.16.2|2.16.3], which can be installed;\n\u2502  \u2502     \u251C\u2500 tiledb [2.16.0|2.16.1] would require\n\u2502  \u2502     \u2502  \u2514\u2500 openssl >=1.1.1u,<1.1.2a , which conflicts with any installable versions previously reported;\n\u2502  \u2502     \u2514\u2500 tiledb [2.16.2|2.16.3] would require\n\u2502  \u2502        \u2514\u2500 openssl >=1.1.1v,<1.1.2a , which conflicts with any installable versions previously reported;\n\u2502  \u251C\u2500 libgdal [3.7.3|3.8.4] would require\n\u2502  \u2502  \u2514\u2500 tiledb >=2.20.1,<2.21.0a0  with the potential options\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.1], which can be installed (as previously explained);\n\u2502  \u2502     \u2514\u2500 tiledb 2.20.1, which can be installed;\n\u2502  \u251C\u2500 libgdal [3.7.3|3.8.4] would require\n\u2502  \u2502  \u2514\u2500 tiledb >=2.21.0,<2.22.0a0 , which can be installed;\n\u2502  \u251C\u2500 libgdal [3.7.3|3.8.3|3.8.4] would require\n\u2502  \u2502  \u2514\u2500 tiledb >=2.20.0,<2.21.0a0  with the potential options\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.0], which can be installed (as previously explained);\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.1], which can be installed (as previously explained);\n\u2502  \u2502     \u251C\u2500 tiledb 2.20.0, which can be installed;\n\u2502  \u2502     \u2514\u2500 tiledb 2.20.1, which can be installed;\n\u2502  \u251C\u2500 libgdal [3.7.3|3.8.4|3.8.5] would require\n\u2502  \u2502  \u2514\u2500 proj >=9.4.0,<9.4.1.0a0 , which does not exist (perhaps a missing channel);\n\u2502  \u251C\u2500 libgdal [3.7.3|3.8.3] would require\n\u2502  \u2502  \u2514\u2500 tiledb >=2.19.1,<2.20.0a0  with the potential options\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.0], which can be installed (as previously explained);\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.1], which can be installed (as previously explained);\n\u2502  \u2502     \u2514\u2500 tiledb 2.19.1, which can be installed;\n\u2502  \u251C\u2500 libgdal [3.7.3|3.8.1] would require\n\u2502  \u2502  \u251C\u2500 proj >=9.3.0,<9.3.1.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2514\u2500 tiledb >=2.18.2,<2.19.0a0  with the potential options\n\u2502  \u2502     \u251C\u2500 tiledb 2.18.2, which can be installed;\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.0], which can be installed (as previously explained);\n\u2502  \u2502     \u251C\u2500 tiledb [2.16.3|2.18.2], which cannot be installed (as previously explained);\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.1], which can be installed (as previously explained);\n\u2502  \u2502     \u2514\u2500 tiledb [2.18.3|2.18.4], which can be installed;\n\u2502  \u251C\u2500 libgdal [3.7.3|3.8.2|3.8.3] would require\n\u2502  \u2502  \u2514\u2500 tiledb >=2.19.0,<2.20.0a0  with the potential options\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.0], which can be installed (as previously explained);\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.1], which can be installed (as previously explained);\n\u2502  \u2502     \u251C\u2500 tiledb 2.19.0, which can be installed;\n\u2502  \u2502     \u2514\u2500 tiledb 2.19.1, which can be installed;\n\u2502  \u251C\u2500 libgdal [3.7.3|3.8.4] would require\n\u2502  \u2502  \u2514\u2500 tiledb >=2.21.1,<2.22.0a0 , which can be installed;\n\u2502  \u251C\u2500 libgdal 3.8.1 would require\n\u2502  \u2502  \u2514\u2500 tiledb >=2.18.2,<2.19.0a0  with the potential options\n\u2502  \u2502     \u251C\u2500 tiledb 2.18.2, which can be installed;\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.0], which can be installed (as previously explained);\n\u2502  \u2502     \u251C\u2500 tiledb [2.16.3|2.18.2], which cannot be installed (as previously explained);\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.1], which can be installed (as previously explained);\n\u2502  \u2502     \u2514\u2500 tiledb [2.18.3|2.18.4], which can be installed;\n\u2502  \u251C\u2500 libgdal 3.8.2 would require\n\u2502  \u2502  \u2514\u2500 tiledb >=2.18.3,<2.19.0a0  with the potential options\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.0], which can be installed (as previously explained);\n\u2502  \u2502     \u251C\u2500 tiledb [2.18.2|2.18.3|...|2.20.1], which can be installed (as previously explained);\n\u2502  \u2502     \u2514\u2500 tiledb [2.18.3|2.18.4], which can be installed;\n\u2502  \u251C\u2500 libgdal [2.2.4|2.3.0] would require\n\u2502  \u2502  \u251C\u2500 hdf5 >=1.10.2,<1.10.3.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2514\u2500 proj4 >=5.0.1,<5.0.2.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u251C\u2500 libgdal [2.2.4|2.3.0] would require\n\u2502  \u2502  \u251C\u2500 hdf5 >=1.8.20,<1.9.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u2502  \u2514\u2500 proj4 >=5.0.1,<5.0.2.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u251C\u2500 libgdal [3.0.2|3.4.1|3.6.0|3.6.2] would require\n\u2502  \u2502  \u2514\u2500 libpq >=12.9,<13.0a0  with the potential options\n\u2502  \u2502     \u251C\u2500 libpq [12.9|13.3|13.5], which can be installed (as previously explained);\n\u2502  \u2502     \u2514\u2500 libpq 12.15 would require\n\u2502  \u2502        \u2514\u2500 krb5 >=1.20.1,<1.21.0a0 , which can be installed;\n\u2502  \u2514\u2500 libgdal 3.6.2 would require\n\u2502     \u2514\u2500 libpq >=12.15,<13.0a0 , which can be installed (as previously explained);\n\u251C\u2500 libpq 16.*  is not installable because there are no viable options\n\u2502  \u251C\u2500 libpq 16.0 would require\n\u2502  \u2502  \u2514\u2500 openssl >=3.1.3,<3.2.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u251C\u2500 libpq 16.1 would require\n\u2502  \u2502  \u251C\u2500 libxml2 >=2.12.2,<3.0.0a0 , which can be installed (as previously explained);\n\u2502  \u2502  \u2514\u2500 openssl >=3.1.4,<3.2.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u251C\u2500 libpq 16.1 would require\n\u2502  \u2502  \u2514\u2500 libxml2 >=2.12.2,<3.0.0a0 , which can be installed (as previously explained);\n\u2502  \u251C\u2500 libpq 16.1 would require\n\u2502  \u2502  \u2514\u2500 openssl >=3.1.4,<3.2.0a0 , which conflicts with any installable versions previously reported;\n\u2502  \u251C\u2500 libpq [16.1|16.2] conflicts with any installable versions previously reported;\n\u2502  \u2514\u2500 libpq 16.3 would require\n\u2502     \u2514\u2500 openssl >=3.3.0,<4.0a0 , which can be installed;\n\u2514\u2500 tiledb 2.22.*  is not installable because it requires\n   \u2514\u2500 libcurl >=8.7.1,<9.0a0 , which requires\n      \u2514\u2500 krb5 >=1.21.2,<1.22.0a0 , which conflicts with any installable versions previously reported.\n                </pre>\n                </div>\n                </details>"
    },
    "pdal_wrench": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": ""
    },
    "postgis": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "",
      "pr_url": "https://github.com/conda-forge/postgis-feedstock",
      "pre_pr_migrator_status": ""
    },
    "pycoalescence": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "unstable",
      "pr_url": "https://github.com/conda-forge/pycoalescence-feedstock/pull/59",
      "pre_pr_migrator_status": ""
    },
    "pygplates": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": "bot error (<a href=\"https://github.com/regro/cf-scripts/actions/runs/9136639068\">bot CI job</a>): main: Traceback (most recent call last):\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 508, in _run_migrator_on_feedstock_branch\n    migrator_uid, pr_json = run(\n                            ^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/auto_tick.py\", line 227, in run\n    rerender_msg = rerender_feedstock(feedstock_dir, timeout=900)\n                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/rerender_feedstock.py\", line 49, in rerender_feedstock\n    return rerender_feedstock_containerized(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/rerender_feedstock.py\", line 102, in rerender_feedstock_containerized\n    data = run_container_task(\n           ^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/cf-scripts/cf-scripts/cf-scripts/conda_forge_tick/utils.py\", line 249, in run_container_task\n    raise RuntimeError(\nRuntimeError: Error running rerender-feedstock in container - error RuntimeError('Failed to rerender.\\noutput: Traceback (most recent call last):\\n  File \"/opt/conda/envs/cf-scripts/bin/conda-smithy\", line 10, in <module>\\n    sys.exit(main())\\n             ^^^^^^\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", line 737, in main\\n    args.subcommand_func(args)\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", line 586, in __call__\\n    self._call(args, args.temporary_directory)\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", line 589, in _call\\n    configure_feedstock.main(\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", line 2695, in main\\n    exclusive_config_file, cf_pinning_ver = get_cached_cfp_file_path(\\n                                            ^^^^^^^^^^^^^^^^^^^^^^^^^\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", line 2516, in get_cached_cfp_file_path\\n    get_cfp_file_path(smithy_cache)\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", line 2440, in get_cfp_file_path\\n    pkg = get_most_recent_version(\"conda-forge-pinning\")\\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", line 2379, in get_most_recent_version\\n    request.raise_for_status()\\n  File \"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/requests/models.py\", line 1021, in raise_for_status\\n    raise HTTPError(http_error_msg, response=self)\\nrequests.exceptions.HTTPError: 524 Server Error:  for url: https://api.anaconda.org/package/conda-forge/conda-forge-pinning\\n\\n') raised:\ncmd: ['docker',\n 'run',\n '-e',\n 'CF_TICK_IN_CONTAINER=true',\n '--security-opt=no-new-privileges',\n '--read-only',\n '--cap-drop=all',\n '--mount',\n 'type=tmpfs,destination=/tmp,tmpfs-mode=1777,tmpfs-size=6000000000',\n '-m',\n '6000m',\n '--cpus',\n '1',\n '--ulimit',\n 'nofile=1024:1024',\n '--ulimit',\n 'nproc=2048:2048',\n '--rm',\n '-i',\n '--mount',\n 'type=bind,source=/tmp/tmpmbs4uaq3,destination=/cf_tick_dir',\n 'ghcr.io/regro/conda-forge-tick:master',\n '/opt/conda/envs/cf-scripts/bin/python',\n '-u',\n '/opt/autotick-bot/docker/run_bot_task.py',\n 'rerender-feedstock',\n '--timeout',\n '900',\n '--log-level',\n 'info']\nstdout: {'data': None,\n 'error': \"RuntimeError('Failed to rerender.\\\\noutput: Traceback (most recent \"\n          'call last):\\\\n  File \"/opt/conda/envs/cf-scripts/bin/conda-smithy\", '\n          'line 10, in <module>\\\\n    sys.exit(main())\\\\n             '\n          '^^^^^^\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", '\n          'line 737, in main\\\\n    args.subcommand_func(args)\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", '\n          'line 586, in __call__\\\\n    self._call(args, '\n          'args.temporary_directory)\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", '\n          'line 589, in _call\\\\n    configure_feedstock.main(\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n          'line 2695, in main\\\\n    exclusive_config_file, cf_pinning_ver = '\n          'get_cached_cfp_file_path(\\\\n                                            '\n          '^^^^^^^^^^^^^^^^^^^^^^^^^\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n          'line 2516, in get_cached_cfp_file_path\\\\n    '\n          'get_cfp_file_path(smithy_cache)\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n          'line 2440, in get_cfp_file_path\\\\n    pkg = '\n          'get_most_recent_version(\"conda-forge-pinning\")\\\\n          '\n          '^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n          'line 2379, in get_most_recent_version\\\\n    '\n          'request.raise_for_status()\\\\n  File '\n          '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/requests/models.py\", '\n          'line 1021, in raise_for_status\\\\n    raise '\n          'HTTPError(http_error_msg, '\n          'response=self)\\\\nrequests.exceptions.HTTPError: 524 Server Error:  '\n          'for url: '\n          \"https://api.anaconda.org/package/conda-forge/conda-forge-pinning\\\\n\\\\n')\",\n 'timeout': 900,\n 'traceback': 'Traceback (most recent call last):\\n'\n              '  File \"/opt/autotick-bot/docker/run_bot_task.py\", line 118, in '\n              '_run_bot_task\\n'\n              '    data = func(**kwargs)\\n'\n              '           ^^^^^^^^^^^^^^\\n'\n              '  File \"/opt/autotick-bot/docker/run_bot_task.py\", line 239, in '\n              '_rerender_feedstock\\n'\n              '    msg = rerender_feedstock_local(fs_dir, **kwargs)\\n'\n              '          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\\n'\n              '  File '\n              '\"/opt/autotick-bot/conda_forge_tick/rerender_feedstock.py\", '\n              'line 239, in rerender_feedstock_local\\n'\n              '    raise RuntimeError(f\"Failed to rerender.\\\\noutput: '\n              '{ret.stdout}\\\\n\")\\n'\n              'RuntimeError: Failed to rerender.\\n'\n              'output: Traceback (most recent call last):\\n'\n              '  File \"/opt/conda/envs/cf-scripts/bin/conda-smithy\", line 10, '\n              'in <module>\\n'\n              '    sys.exit(main())\\n'\n              '             ^^^^^^\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", '\n              'line 737, in main\\n'\n              '    args.subcommand_func(args)\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", '\n              'line 586, in __call__\\n'\n              '    self._call(args, args.temporary_directory)\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/cli.py\", '\n              'line 589, in _call\\n'\n              '    configure_feedstock.main(\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n              'line 2695, in main\\n'\n              '    exclusive_config_file, cf_pinning_ver = '\n              'get_cached_cfp_file_path(\\n'\n              '                                            '\n              '^^^^^^^^^^^^^^^^^^^^^^^^^\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n              'line 2516, in get_cached_cfp_file_path\\n'\n              '    get_cfp_file_path(smithy_cache)\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n              'line 2440, in get_cfp_file_path\\n'\n              '    pkg = get_most_recent_version(\"conda-forge-pinning\")\\n'\n              '          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/conda_smithy/configure_feedstock.py\", '\n              'line 2379, in get_most_recent_version\\n'\n              '    request.raise_for_status()\\n'\n              '  File '\n              '\"/opt/conda/envs/cf-scripts/lib/python3.11/site-packages/requests/models.py\", '\n              'line 1021, in raise_for_status\\n'\n              '    raise HTTPError(http_error_msg, response=self)\\n'\n              'requests.exceptions.HTTPError: 524 Server Error:  for url: '\n              'https://api.anaconda.org/package/conda-forge/conda-forge-pinning\\n'\n              '\\n'\n              '\\n'}\n"
    },
    "pyogrio": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9143944538'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_ppc64le_python3.10.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_python3.11.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_python3.12.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_python3.8.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_python3.9.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).\n                </pre>\n                </div>\n                </details>"
    },
    "qgis": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": ""
    },
    "r-gdalcubes": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "",
      "pr_url": "https://github.com/conda-forge/r-gdalcubes-feedstock",
      "pre_pr_migrator_status": ""
    },
    "r-rgdal": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9144467442'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_ppc64le_r_base4.2: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_r_base4.3: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).\n                </pre>\n                </div>\n                </details>"
    },
    "r-sf": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9143403814'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_ppc64le_r_base4.2: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_r_base4.3: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).\n                </pre>\n                </div>\n                </details>"
    },
    "r-terra": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9142796336'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_ppc64le_r_base4.2: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_r_base4.3: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).\n                </pre>\n                </div>\n                </details>"
    },
    "rasam": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "clean",
      "pr_url": "https://github.com/conda-forge/rasam-feedstock/pull/2",
      "pre_pr_migrator_status": ""
    },
    "rasterio": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9141171382'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_ppc64le_numpy1.22python3.10.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.22python3.8.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.22python3.9.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.23python3.11.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.26python3.12.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 proj 9.4.0.*  does not exist (perhaps a typo or a missing channel).\n                </pre>\n                </div>\n                </details>"
    },
    "richdem": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9144467442'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_ppc64le_numpy1.22python3.10.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.22python3.8.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.22python3.9.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.23python3.11.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.26python3.12.____cpython: Could not solve for environment specs\nThe following package could not be installed\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).\n                </pre>\n                </div>\n                </details>"
    },
    "rsgislib": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "",
      "pr_url": "https://github.com/conda-forge/rsgislib-feedstock",
      "pre_pr_migrator_status": ""
    },
    "taudem": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "clean",
      "pr_url": "https://github.com/conda-forge/taudem-feedstock/pull/18",
      "pre_pr_migrator_status": ""
    },
    "tuiview": {
      "immediate_children": [],
      "num_descendants": 0,
      "pre_pr_migrator_status": "not solvable (<a href='https://github.com/regro/cf-scripts/actions/runs/9144467442'>bot CI job</a>) @ main\n                <details>\n                <div align=\"left\">\n                <pre>\n                linux_ppc64le_numpy1.22python3.10.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 gdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.22python3.8.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 gdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.22python3.9.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 gdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.23python3.11.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 gdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).</pre><pre>linux_ppc64le_numpy1.26python3.12.____cpython: Could not solve for environment specs\nThe following packages are incompatible\n\u251C\u2500 gdal 3.9.*  does not exist (perhaps a typo or a missing channel);\n\u2514\u2500 libgdal 3.9.*  does not exist (perhaps a typo or a missing channel).\n                </pre>\n                </div>\n                </details>"
    },
    "xtensor-zarr": {
      "immediate_children": [],
      "num_descendants": 0,
      "pr_status": "clean",
      "pr_url": "https://github.com/conda-forge/xtensor-zarr-feedstock/pull/14",
      "pre_pr_migrator_status": ""
    }
  },
  "awaiting-parents": [
    "qgis",
    "pdal_wrench"
  ],
  "awaiting-pr": [],
  "bot-error": [
    "pygplates"
  ],
  "done": [
    "rsgislib",
    "r-gdalcubes",
    "postgis",
    "pcraster",
    "ncl",
    "mesher",
    "mdal",
    "helios",
    "gdal-csharp",
    "gazebo"
  ],
  "in-pr": [
    "heavydb-ext",
    "xtensor-zarr",
    "taudem",
    "rasam",
    "pycoalescence",
    "nansat",
    "lue",
    "isce3",
    "isce2",
    "hexer",
    "exactextract",
    "dggrid",
    "autorift",
    "acctaudem"
  ],
  "not-solvable": [
    "tuiview",
    "richdem",
    "rasterio",
    "r-terra",
    "r-sf",
    "r-rgdal",
    "pyogrio",
    "pdal",
    "mapserver",
    "gz-common",
    "gmt",
    "fiona"
  ]
}